commit 871f9934fa0ea25906e338a74d02bb361f6490bf
Author: Dan Williams <dcbw@redhat.com>
Date:   Mon Apr 26 23:56:01 2010 -0700

    build: default to pppd-2.4.5 plugin dir
    
    People should be using 2.4.5 now anyway, since it's been out for
    quite a while and includes the fixes for the bogus DNS servers.

diff --git a/configure.ac b/configure.ac
index adfe67f..5918c26 100644
--- a/configure.ac
+++ b/configure.ac
@@ -296,7 +296,7 @@ AC_ARG_WITH([pppd-plugin-dir], AS_HELP_STRING([--with-pppd-plugin-dir=DIR], [pat
 if test -n "$with_pppd_plugin_dir" ; then
 	PPPD_PLUGIN_DIR="$with_pppd_plugin_dir"
 else
-	PPPD_PLUGIN_DIR="${libdir}/pppd/2.4.4"
+	PPPD_PLUGIN_DIR="${libdir}/pppd/2.4.5"
 fi
 AC_SUBST(PPPD_PLUGIN_DIR)
 
